AccountController.rb

```diff
@@ -0,0 +1,65 @@
+require_relative 'FileHandling'
+require_relative 'validation'
+require_relative 'account'
+
+module AccountController
+  include Validation
+
+  def start_account_controller
+    @file_system = FileSystem.new
+    @file_system.load_account_data
+    @account_data = @file_system.account_data
+    @account_numbers = @file_system.account_numbers
+    puts "this is account data loaded: #{@account_data}"
+  end
+
+  def store_account_data
+    @file_system.update_account_files(@account_data)
+  end
+
+  def create_account(account_number, pin, email)
+    Account.new(account_number, pin, email)
+  end
+
+  def add_account(account)
+    @account_data[account.email] ||= []
+    @account_data[account.email] << account
+  end
+
+  def generate_account_number
+    account_number = 0
+    loop do
+      account_number = rand(10000...99999).to_s
```

Comment:
why is account number random?

Reply:
I wanted the account numbers to be within a certain range and also banks don't give you an option to set your own account number, they just generate it randomly.